client/src/components/MapView/useCoordinateResolver.ts

```diff
@@ -0,0 +1,30 @@
+import { Location } from '../../gql/graphql.ts';
+import { useQuayCoordinateQuery } from '../../hooks/useQuayCoordinateQuery.ts';
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+export function useCoordinateResolver(location: Location): Coordinates | undefined {
+  const quay = useQuayCoordinateQuery(location);
+
+  if (quay) {
+    const { longitude, latitude } = quay;
+
+    if (longitude && latitude) {
+      return {
+        longitude,
+        latitude,
```

Comment:
My knowledge of JS is limited - but it looks like the arguments order is swapped according to the  interface definition above. As a practice to avoid errors on this we always do `lat, long` in the Java code never `long, lat`.

Reply:
I changed the order of the params to follow the convention. (Although it wasn't a bug, because I used here the JavaScript's [property shorthand syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions)).